RecoTracker/TkSeedGenerator/plugins/SeedGeneratorFromL1TTracksEDProducer.cc

```diff
@@ -61,7 +62,7 @@ class dso_hidden SeedGeneratorFromL1TTracksEDProducer : public edm::global::EDPr
   const double theErrorSFHitless_;
 
   const edm::ESGetToken<MagneticField, IdealMagneticFieldRecord> mfToken_;
-  const edm::ESGetToken<TrackerGeometry, TrackerRecoGeometryRecord> geomToken_;
+  const edm::ESGetToken<TrackerGeometry, TrackerDigiGeometryRecord> geomToken_;
```

Comment:
this was a part changed during the review of RecoTracker/TkSeedGenerator/plugins/SeedGeneratorFromL1TTracksEDProducer.cc
At least in this case it appeared to be OK to use DigiGeometry because this is used only to check presence of some elements (not their actual positioning).
Still, I wanted to check if this change here was intentional and not a result of picking up an outdated version of the file.
Please confirm.

Reply:
This actually is not _completely_ intentional since I was testing to understand it better it but given it was raised here: not an expert here but the `TrackerGeometry` object isn't _stored_ in the `TrackerRecoGeometryRecod`, is it?  If you think is still better to switch to RecoGeometry the some extra changes need to be done (e.g. [here](https://github.com/cms-sw/cmssw/pull/30817/files/1c1c31e50949b30275f34be26236e86677ce8899#diff-86e45c8bc92c5c9b1a5a7fbc2f5191a3R158)). Am I wrong?

P.S. The change in the PR wasn't actually tested ...